Source/Particles/Collision/CollisionType.cpp

```diff
@@ -134,7 +132,11 @@ void CollisionType::doCoulombCollisionsWithinTile
 
         const Real dt = WarpX::GetInstance().getdt(lev);
         Geometry const& geom = WarpX::GetInstance().Geom(lev);
-        const Real dV = geom.CellSize(0)*geom.CellSize(1)*geom.CellSize(2);
+        #if (AMREX_SPACEDIM == 2)
+        auto dV = geom.CellSize(0) * geom.CellSize(1);
```

Comment:
General question: in 2D3V are all our quantities somehow line-densities? or are we normalizing the cell in y to be 1m?

Reply:
From Remi and Maxence, I believe the weights have units (m^-1). So, it is really 2D with line-densities and infinite y.

Reply:
Urgh, that always breaks my head -.-
I prefer pseudo-scale to an arbitrary value, e.g. same as dx or dz...

Reply:
Okay, just checked with @MaxThevenet again.
Since our input and output units are numerically the same in 3D3V and 2D3V, this is fine. One can interpret that as two things:
- units are the same, dy is implicitly 1m or
- all input and input units are implictly *1m^-1 as unit dimensionality in 2D3V

Both ways allow skipping the neutral element in ops like these.